tasks/test_syntax.js

```diff
@@ -70,9 +70,16 @@ function assertSrcContents() {
                 // look for .classList
                 if(node.type === 'MemberExpression') {
                     var parts = node.source().split('.');
-                    if(parts[parts.length - 1] === 'classList') {
+                    var lastPart = parts[parts.length - 1];
+                    if(lastPart === 'classList') {
                         logs.push(file + ' : contains .classList (IE failure)');
                     }
+                    else if(lastPart === 'innerHTML') {
+                        logs.push(file + ' : contains .innerHTML (IE failure in SVG)');
```

Comment:
There might be exceptions to this rule in the future, if we end up using `innerHTML` on non-SVG DOM elements. Oh well, we could always filter the cases out then. We might we want to add a comment here signalling this check is too strict.